inc/Config/Datasource/CompatibleHttpDatasource.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace RemoteDataBlocks\Config\Datasource;
+
+/**
+ * HttpDatasource class
+ *
+ * Implements the HttpDatasourceInterface to define a generic HTTP datasource.
+ *
+ * @package remote-data-blocks
+ * @since 0.1.0
+ */
+abstract class CompatibleHttpDatasource extends HttpDatasource {
```

Comment:
```suggestion
abstract class HardcodeHttpDatasource extends HttpDatasource {
```

Whatcha think? Not elegant, but, to the point. 

"Compatible" doesn't give me much to hang my head on mentally.

Reply:
I hear that, "compatible" is sort of inherent in our use of interfaces, so it's not a good descriptor. Let's get some wider feedback and I'll make the edit.